pkg/blobstore/local/hashing_digest_location_map.go

```diff
@@ -165,18 +168,25 @@ func (dlm *hashingDigestLocationMap) Put(digest digest.Digest, validator *Locati
 	}
 	for iteration := 1; iteration <= dlm.maximumPutAttempts; iteration++ {
 		slot := dlm.getSlot(&record.Key)
-		oldRecord := dlm.recordArray.Get(slot)
+		oldRecord, err := dlm.recordArray.Get(slot)
+		if err != nil {
+			return err
```

Comment:
So far all of these code paths were written to increment Prometheus histograms. We don't do this for the error cases. I guess that's fine, because we already provide BlobAccess error metrics? No need to change anything; simply braindumping!